xml_inscriptions/transcribed/MI.AA.UM.KM.L.968.xml

```diff
@@ -111,27 +118,27 @@
                <!--  <app type="previouslyread">
                   <lem resp="previous">-->
                         <expan>
-                           <abbr><supplied reason="undefined" evidence="previouseditor">D</supplied></abbr><ex>is</ex>
+                           <abbr><supplied reason="lost" evidence="previouseditor">D</supplied></abbr><ex>is</ex>
```

Comment:
The change of "undefined" to "lost" here isn't particularly important, especially as the stylesheets will handle them identically using our output style. Certain other projects are customized to add other material when you handle it this way.